### DISCUSSION TITLE
Showcase: PDF Step & Repeat

### DISCUSSION THREAD

**Discussion by felle9900** [TomSchimansky/CustomTkinter#145] [Showcase]
Hi Guys, just wanted to share a app I've made using CustomTkinter as the guy.

The App is a tool for placing multiple pages of a pdf onto a new pdf, often used in printshops and prepress.
It support sequential numbers and variable data (placing it as Cut & Stack).

I hope you like it - And thank you Tom for creating this Awesome gui !

https://www.youtube.com/watch?v=OY9QGf644eo

**Comment by ghost** [comment]
Stunning! Great work. 👍🏻

**Reply by felle9900** [reply]
Thanks a lot :)

**Comment by sumeshir26** [comment]
Nice :)

**Reply by felle9900** [reply]
Thanks ;)

**Comment by kelltom** [comment]
Woah, nice. I thought there were some issues with CustomTkinter on Mac? Or did I just dream that?

**Reply by felle9900** [reply]
Well there probably are some issues, but bo big issues for me on a mac. 

Only the auto system dark/normal mode is not working for me, but there's a fix for that somewhere I read. Not important for me at this point.

**Reply by TomSchimansky** [reply]
I develop CustomTkinter on macOS, everything should run fine. Actually, the rendering looks better than on Windows and Linux because the tkinter canvas is antialiased on macOS be default and scaling also works automatically.

**Reply by kelltom** [reply]
This is awesome to hear! I must've imagined it lol.

**Reply by felle9900** [reply]
I se ur working on some stuff but the changelog haven't been updated in a while.
It it because it's just minor tweaks you are doing ?

**Reply by TomSchimansky** [reply]
Yeah the latest changes were all mostly fixes or small upgrades. The next minor version upgrade 4.6 will be when the CTkTextbox is ready and this will appear in the changelog.
